src/ui/public/state_management/state.js

```diff
@@ -24,12 +24,16 @@ export default function StateProvider(Private, $rootScope, $location) {
         self.fetch();
       }),
 
+      $rootScope.$on('$routeChangeSuccess', function () {
```

Comment:
⚠️ super nit approaching ⚠️ 

can you swap these event handlers so that start comes before success?